app/components/JenkinsJobs.tsx

```diff
@@ -28,6 +28,17 @@ const jenkinsColourToWebDashColour = new Map<string, string>([
   ], // build aborted but now running
 ]);
 
+const jenkinsColourToStatus = new Map<string, string>([
+  ["red", "fail"], // build broken
+  ["blue", "success"], // build success
+  ["aborted", "aborted"], // build aborted
+  ["yellow", "unstable"],
+  ["yellow_anime", "unstable-running"], // build running but was broken
+  ["red_anime", "fail-running"], // build running but was broken
+  ["blue_anime", "success-running"], // build running but was successful
+  ["aborted_anime", "aborted-running"], // build aborted but now running
```

Comment:
Is there a better form of wording for `fail-running`, `unstable-running` etc?

success -> succeeded
fail -> failed

`running (failed)` vs `failed - running`?